e2e/features/compare/compare-mobile-test.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
 
   // load SWIPE and verify mobile compare A|B toggle buttons are visible
   'Mobile comparison A|B toggle buttons are visible and only A is selected by default': (c) => {
+    c.pause(10000);
```

Comment:
This seems like a really long pause, is it necessary?

Reply:
Removed pause, this was left over from debugging.